src/main/java/io/jenkins/plugins/pipelinegraphview/utils/AbstractPipelineViewAction.java

```diff
@@ -41,10 +43,18 @@ public boolean isBuildable() {
         return run.getParent().isBuildable();
     }
 
+    public boolean isBuilding() {
+        return run.isBuilding();
+    }
+
     public Permission getPermission() {
         return run.getParent().BUILD;
     }
 
+    public Permission getCancelPermission() {
+        return run.getParent().CANCEL;
```

Comment:
intellij warning here, may as well not make it any worse:
```suggestion
        return Item.CANCEL;
```